src/providers/api/api.ts

```diff
@@ -0,0 +1,25 @@
+export class ApiProvider {
+  isProduction = true;
+  private config = {
+    prod: {
+      bitcore: 'https://ducws.rocknblock.io',
+      ducatuscoins: 'https://www.ducatuscoins.com',
+      crowdsale: 'https://tokenization.centuriongm.com',
+      pog: 'https://d-pog.com'
+    },
+    develop: {
+      bitcore: 'https://duc-ws-dev.rocknblock.io',
+      ducatuscoins: 'https://ducsite.rocknblock.io',
+      crowdsale: 'http://duccrowdsale.rocknblock.io',
+      pog: 'https://devgold.rocknblock.io'
```

Comment:
Будет ошибка, надо добавить еще один параметр для ratesApi в файле 
```suggestion
      pog: 'https://devgold.rocknblock.io',
      ratesApi: 'https://ducexpl.rocknblock.io/api/v1/rates/'
```